### PR TITLE
Avoid access key logging during BR/BRC workflows

### PR DESCRIPTION
**Pull request by deepakkinni**
1. Remove log statements.

Signed-off-by: Deepak Kinni <dkinni@vmware.com>

### CODE REVIEW

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/285/

File: pkg/backuprepository/backup_repository.go [hunk 1]
```diff
@@ -210,8 +210,8 @@ func checkIfBackupRepositoryClaimIsReferenced(
 			}
 		}
 	} else {
-		logger.Infof("Received BackupRepositoryClaim with no BackupRepository reference, ignoring it now, as it"+
-			"is expected to be patched later, BackupRepositoryClaim: %v", backupRepositoryClaim)
+		logger.Infof("Received BackupRepositoryClaim: %s with no BackupRepository reference, ignoring it now, as it"+
```
Comment (xing-yang) [inline pkg/backuprepository/backup_repository.go:213]: Print namespace/name

File: pkg/backuprepository/backup_repository.go [hunk 2]
```diff
@@ -267,7 +267,7 @@ func compareBackupRepositoryClaim(repositoryDriver string,
 	backupRepositoryClaim *backupdriverv1.BackupRepositoryClaim,
 	logger logrus.FieldLogger) bool {
 	// Compare the params. RepositoryDriver is always same.
-	logger.Infof("Comparing BRC: %v", backupRepositoryClaim)
+	logger.Infof("Comparing BRC: %s", backupRepositoryClaim.Name)
```
Comment (xing-yang) [inline pkg/backuprepository/backup_repository.go:270]: Add namespace

File: pkg/backuprepository/backup_repository.go (hunk 1 above)
Comment (deepakkinni) [reply pkg/backuprepository/backup_repository.go]: Done.

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/286/

**Comment by xing-yang** [comment]
LGTM

**Pull request merged**